src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -820,6 +979,7 @@ public void onSearch(@Nullable String query) {
 	public void onRefreshPosts() {
 		postListingController.setSession(null);
 		requestRefresh(RefreshableFragment.POSTS, true);
+		mTabLayout.getTabAt(activeTab).select();
```

Comment:
Is this being called to trigger some sort of `onTabSelected()` callback somewhere? If so, is this guaranteed by the API docs to work even when that tab is already selected?

Reply:
(in any case, I'm currently of the opinion that the tabs could be removed, and we could just rely on the drawer)